tests/src/Unit/settingsTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Testing the settings: search_config_node_settings()
+ */
+use \Drupal\module;
```

Comment:
The namespace would be Drupal\search_config not module b/c it should be the module name.